clusterloader2/testing/network/config.yaml

```diff
@@ -0,0 +1,40 @@
+  {{$Replicas := .Replicas}}
+  {{$Ratio := .Ratio}}
+  {{$Protocol := .Protocol}}
+name: network
+automanagedNamespaces: 1
+tuningSets:
+- name: Sequence
+  parallelismLimitedLoad:
+    parallelismLimit: 1
+steps:
+- name: start network metric measurement
+  measurements:
+    - Identifier: NetworkPerformanceMetrics
+      Method: NetworkPerformanceMetrics
+      Params:
+        action: start 
+        podReplicas: {{$Replicas}}
+        duration: 10
+        ratio: {{$Ratio}} 
+        protocol: {{$Protocol}}
+- name: Wait for 15 secs controller to be up
```

Comment:
That's not needed anymore, right?

Reply:
Its requred. After calling start() , if we dont have wait in-between , gather() is called immediately. In gather() we read the result and return immediately. We dont wait or do retry in gather() call.